mapie/regression/__init__.py

```diff
@@ -1,9 +1,11 @@
 from .regression import MapieRegressor
 from .quantile_regression import MapieQuantileRegressor
 from .time_series_regression import MapieTimeSeriesRegressor
+from .estimator import EnsembleRegressor
 
 __all__ = [
     "MapieRegressor",
     "MapieQuantileRegressor",
-    "MapieTimeSeriesRegressor"
+    "MapieTimeSeriesRegressor",
+    "EnsembleRegressor"
```

Comment:
UPD: Shouldn't `EnsembleRegressor` be accessible to the user, am I wrong?

Reply:
You want it to be inaccessible?